src/view/frontend/templates/alternate_urls.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Elgentos\AlternateUrls\Block\AlternateUrls;
+use Elgentos\AlternateUrls\Type\TypeInterface;
+use Magento\Framework\Escaper;
+
+/** @var AlternateUrls $block */
+/** @var Escaper $escaper */
+
+/** @var TypeInterface $typeInstance */
+$typeInstance = $block->getTypeInstance();
```

Comment:
You are assuming the type of this variable above this line, and then checking against falsyness a couple lines later.

Consider the following:

```php

$typeInstance = $block->getTypeInstance();

if (!$typeInstance instanceof TypeInterface) {
    return;
}

foreach ($typeInstance->getAlternateUrls() as $alternateUrl):
    $hreflang = $escaper->escapeHtmlAttr($alternateUrl['hreflang']);
    $href = $escaper->escapeUrl($alternateUrl['url']);
?>
    <link rel="alternate" hreflang="<?= $hreflang ?>" href="<?= $href ?>">
<?php endforeach ?>
```

Reply:
See also: https://github.com/elgentos/magento2-alternate-urls/pull/1#pullrequestreview-800961656